frank/plot.py

```diff
@@ -336,36 +331,25 @@ def plot_2dsweep(r, I, ax, cmap='inferno', norm=None, vmin=None,
                                                     project=project,
                                                     phase_shift=phase_shift,
                                                     geom=geom)
+
     if xmax is None:
         xmax = xmax_computed
     if ymax is None:
         ymax = ymax_computed
 
-    if rescale_brightness:
-        I2D /= 1e10
-        vmin /= 1e10
-        vmax /= 1e10
-
-    if vmin is None:
-        vmin = I2D.min()
-    if vmax is None:
-        vmax = I2D.max()
+    if norm is None:
+        import matplotlib.colors as mpl_cs
+        norm = mpl_cs.Normalize(vmin=I2D.min(), vmax=I2D.max())
```

Comment:
I think it would be better to leave norm as None. I suspect that its equivalent while probably handling NaN's more elegantly (although they should not occur).